conf/evaluator_common.env

```diff
@@ -1,2 +1,3 @@
 DB_USER=evaluator
 DB_PASSWD=evaluator
+VMAAS_CALL_COMPRESSION=on
```

Comment:
We should follow naming convention where booleans have `ENABLE` in the name

Reply:
updated